$NetBSD: patch-aa,v 1.25 2017/04/11 09:33:30 adam Exp $

Define LD_LIBRARY_PATH in this script as it is not propagated from outside.
Make this pkgsrc friendly.

--- postfix-install.orig	2016-01-03 00:25:23.000000000 +0000
+++ postfix-install
@@ -199,6 +199,7 @@ SHELL=/bin/sh
 IFS=" 	
 "
 BACKUP_IFS="$IFS"
+LD_LIBRARY_PATH=`pwd`/lib; export LD_LIBRARY_PATH
 
 # This script uses outputs from Postfix and non-Postfix commands.
 # Override all LC_* settings and LANG for robustness.
@@ -447,26 +448,6 @@ template files main.cf.proto and master.
 : ${tempdir=`pwd`}
 : ${config_directory=`bin/postconf -c conf -h -d config_directory`}
 
-# Find out the location of installed configuration files.
-
-test -z "$non_interactive" && for name in install_root tempdir config_directory
-do
-    while :
-    do
-	echo
-	eval echo Please specify \$${name}_prompt | ${FMT}
-	eval echo \$n "$name: [\$$name]\  \$c"
-	read ans
-	case $ans in
-	"") break;;
-	 *) case $ans in
-	    /*) eval $name=$ans; break;;
-	     *) echo; echo $0: Error: $name should be an absolute path name. 1>&2;;
-	    esac;;
-	esac
-    done
-done
-
 # In case some systems special-case pathnames beginning with //.
 
 case $install_root in
@@ -869,13 +850,3 @@ bin/postconf -c $CONFIG_DIRECTORY -e \
     "shlib_directory = $shlib_directory" \
     "meta_directory = $meta_directory" \
 || exit 1
-
-# If Postfix is being installed locally from source code, do the
-# post-install processing now.
-
-# The unexpansion above may have side effects on exported variables.
-# It does not matter because bin/postfix below will override them.
-
-test -n "$install_root" || {
-    bin/postfix post-install $post_install_options || exit 1
-}
